src/graph/AddSchemaFromSpaceExecutor.cpp

```diff
@@ -0,0 +1,70 @@
+/* Copyright (c) 2020 vesoft inc. All rights reserved.
+ *
+ * This source code is licensed under Apache 2.0 License,
+ * attached with Common Clause Condition 1.0, found in the LICENSES directory.
+ */
+
+#include "graph/AddSchemaFromSpaceExecutor.h"
+
+namespace nebula {
+namespace graph {
+
+AddSchemaFromSpaceExecutor::AddSchemaFromSpaceExecutor(Sentence *sentence,
+                                         ExecutionContext *ectx)
+        : Executor(ectx, "add_schema_from_space") {
+    sentence_ = static_cast<AddSchemaFromSpaceSentence*>(sentence);
+}
+
+
+Status AddSchemaFromSpaceExecutor::prepare() {
+    spaceName_ = sentence_->spaceName();
+    if (spaceName_ == nullptr) {
+        return Status::Error("Empty space name");
+    }
+    return Status::OK();
+}
+
+void AddSchemaFromSpaceExecutor::execute() {
+    auto status = checkIfGraphSpaceChosen();
+    if (!status.ok()) {
+        doError(std::move(status));
+        return;
+    }
+    auto currentSpace = ectx()->rctx()->session()->spaceName();
+    if (currentSpace == *spaceName_) {
+        LOG(ERROR) << "Current space `" << currentSpace << "' as same as from space `"
```

Comment:
' as same as from space ` I'm confused here. Do you mean ` the same as the source space`?